src/sentry/static/sentry/app/stores/hookStore.jsx

```diff
@@ -24,6 +24,8 @@ const validHookNames = new Set([
   // Specific component customizations
   'component:org-auth-view',
   'component:org-members-view',
+  'component:date-range',
+  'component:selector-list',
```

Comment:
Would be nice if we could give these a slightly more sentry-specific name:

```
component:header-date-range
component:header-date-selector-list
```

maybe?